.github/CODEOWNERS

```
@@ -316,3 +316,13 @@ Lib/test/test__colorize.py    @hugovk
 
 # Fuzzing
 Modules/_xxtestfuzz/          @ammaraskar
+
+# t-strings
+Include/internal/pycore_interpolation.h  @lysnikolaou
+Include/internal/pycore_template.h  @lysnikolaou
+Objects/interpolationobject.c  @lysnikolaou
+Objects/clinic/interpolationobject.c.h  @lysnikolaou
+Objects/templateobject.c      @lysnikolaou
+Modules/_templatelibmodule.c  @lysnikolaou
+Lib/string/templatelib.py     @lysnikolaou
+Lib/test/test_tstring.py      @lysnikolaou
```

Sort 'em
```suggestion
Lib/string/templatelib.py     @lysnikolaou
Lib/test/test_tstring.py      @lysnikolaou
Modules/_templatelibmodule.c  @lysnikolaou
Objects/clinic/interpolationobject.c.h  @lysnikolaou
Objects/interpolationobject.c  @lysnikolaou
Objects/templateobject.c      @lysnikolaou
```

---

Another alternative:

```suggestion
**/*interpolationobject*      @lysnikolaou
**/*templatelib*              @lysnikolaou
**/*templateobject*           @lysnikolaou
**/*tstring.py                @lysnikolaou
```